fe/fe-core/src/main/java/com/starrocks/catalog/Type.java

```diff
@@ -133,23 +133,28 @@ public abstract class Type implements Cloneable {
     public static final Type ARRAY_VARCHAR = new ArrayType(Type.VARCHAR);
     public static final Type ARRAY_JSON = new ArrayType(Type.JSON);
 
-    private static final ImmutableList<ScalarType> INTEGER_TYPES =
-            ImmutableList.of(TINYINT, SMALLINT, INT, BIGINT, LARGEINT);
-    private static final ImmutableList<ScalarType> FLOAT_POINT_TYPES =
-            ImmutableList.of(FLOAT, DOUBLE, DECIMALV2, DECIMAL32, DECIMAL64, DECIMAL128);
+    public static final ImmutableList<ScalarType> INTEGER_TYPES =
+            ImmutableList.of(BOOLEAN, TINYINT, SMALLINT, INT, BIGINT, LARGEINT);
```

Comment:
why change `BOOLEAN` ?

Reply:
It's just like the behavior in `BE`, `BE` actually is UINT8Column?